migrations/sql/V2019.07.24.11.55__add_tables_to_stage_NROS_and_VFCBC_data.sql

```diff
@@ -0,0 +1,494 @@
+CREATE SCHEMA IF NOT EXISTS now;
```

Comment:
As discussed, I think we pivoted a little on our implementation. This schema won't hold NOW's but will hold NOW Submissions. Can we name the scheme to NOW_Submissions.

Reply:
Changed to now_submissions